includes/SpecialRemovePII.php

```diff
@@ -15,7 +15,6 @@
 use MediaWiki\Extension\CentralAuth\GlobalRename\GlobalRenameUserValidator;
 use MediaWiki\Extension\CentralAuth\User\CentralAuthUser;
 use MediaWiki\JobQueue\JobQueueGroupFactory;
-use MediaWiki\User\UserFactory;
```

Comment:
Why remove this? You should remove CentralAuthUser (the non-namespaced one)

Reply:
I thought I added the namespaced one

Reply:
You did. You need to remove the first one or else it is a confliction.